src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -318,6 +328,12 @@ export const mapRouteError: MapRouteError = (error) => {
   }
 }
 
+/**
+ * Checks whether attachmentMap contains the given response
+ * @param attachmentMap Map of field IDs to attachments
+ * @param response The response to check
+ * @returns true if response is in map, false otherwise
+ */
```

Comment:
I know this is outside the scope of this PR, but are we doing insufficient checks for this type guard? I was surprised to see `isAttachmentResponseFromMap` merely check for the existence of a value from `attachmentMap: Record<IAttachmentInfo['fieldId'], IAttachmentInfo>`. Would that be sufficient?

For reference:

```typescript
export interface IAttachmentInfo {
  filename: string
  content: Buffer
  fieldId: string
}

export interface IAttachmentResponse extends IBaseResponse {
  fieldType: BasicField.Attachment
  filename: string
  content: Buffer
  answer: string
}
```

Reply:
good idea, this can be tightened up